diff --git a/node_modules/react-native-markdown-display/package.json b/node_modules/react-native-markdown-display/package.json
index 0670296..93bc4b5 100644
--- a/node_modules/react-native-markdown-display/package.json
+++ b/node_modules/react-native-markdown-display/package.json
@@ -35,7 +35,7 @@
     "react-native-fit-image": "^1.5.5"
   },
   "peerDependencies": {
-    "react": "^16.2.0",
+    "react": ">=16.2.0",
     "react-native": ">=0.50.4"
   },
   "devDependencies": {
diff --git a/node_modules/react-native-markdown-display/src/index.d.ts b/node_modules/react-native-markdown-display/src/index.d.ts
index f0daf4a..0e46a73 100644
--- a/node_modules/react-native-markdown-display/src/index.d.ts
+++ b/node_modules/react-native-markdown-display/src/index.d.ts
@@ -1,7 +1,7 @@
 // tslint:disable:max-classes-per-file
 import MarkdownIt from 'markdown-it';
 import Token from 'markdown-it/lib/token';
-import {ComponentType, ReactNode} from 'react';
+import React, {ComponentType, ReactNode} from 'react';
 import {StyleSheet, View} from 'react-native';
 
 export function getUniqueID(): string;
@@ -82,6 +82,7 @@ export function stringToTokens(
 export function tokensToAST(tokens: ReadonlyArray<Token>): ASTNode[];
 
 export interface MarkdownProps {
+  children?: React.ReactNode;
   rules?: RenderRules;
   style?: StyleSheet.NamedStyles<any>;
   renderer?: AstRenderer;
diff --git a/node_modules/react-native-markdown-display/src/lib/util/tokensToAST.js b/node_modules/react-native-markdown-display/src/lib/util/tokensToAST.js
index b0ed265..4b6a6a1 100644
--- a/node_modules/react-native-markdown-display/src/lib/util/tokensToAST.js
+++ b/node_modules/react-native-markdown-display/src/lib/util/tokensToAST.js
@@ -61,6 +61,7 @@ export default function tokensToAST(tokens) {
       )
     ) {
       astNode.index = children.length;
+      astNode.key= `${astNode.type}-${astNode.index}`
 
       if (token.nesting === 1) {
         children.push(astNode);
