diff --git a/node_modules/react-native-reanimated/react-native-reanimated.d.ts b/node_modules/react-native-reanimated/react-native-reanimated.d.ts
index a153973..45bca3c 100644
--- a/node_modules/react-native-reanimated/react-native-reanimated.d.ts
+++ b/node_modules/react-native-reanimated/react-native-reanimated.d.ts
@@ -154,7 +154,7 @@ declare module 'react-native-reanimated' {
       getNode(): ReactNativeFlatList;
     }
     // eslint-disable-next-line @typescript-eslint/no-empty-interface
-    export interface FlatList<T> extends ReactNativeView<T> {}
+    export interface FlatList<T> extends ReactNativeFlatList<T> {}
 
     type Options<P> = {
       setNativeProps: (ref: any, props: P) => void;
